app/components/__tests__/FormLogin.test.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import React from "react";
+import { shallow } from "enzyme";
+
+import navigationMock from "mocks";
+
+import { FormLogin } from "../FormLogin";
+
+describe("Loading", () => {
+  it("has a valid snapshot", () => {
+    const component = shallow(<FormLogin navigation={navigationMock} />);
```

Comment:
Aha I'd not seen snapshot rendering done with Enzyme before. I've always used [`react-test-renderer`](https://www.npmjs.com/package/react-test-renderer). I guess one difference to be aware of is that this is shallow rendering (I don't think Enzyme supports deep rendering with React Native) so will only render one component layer deep, whereas `react-test-renderer` is deep rendering.